TFDS_Python_Interoperability.ipynb

```diff
@@ -0,0 +1,342 @@
+            "text/plain": [
+              "<Figure size 432x288 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-GMcqFWUElb-",
+        "colab_type": "text"
+      },
+      "source": [
+        "Create training batches:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "5YOJ2vGPkDBI",
+        "colab_type": "code",
+        "outputId": "623d0f2d-ce85-44b7-b1c9-59ce168d131c",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 90
+        }
+      },
+      "source": [
+        "let ds_train = mnist.shuffle(1000).batch(128).prefetch(10)\n",
+        "for features in ds_train.take(3) {\n",
+        "  let image = features[\"image\"], label = features[\"label\"]\n",
+        "  print(image.shape, label.shape)\n",
+        "}\n",
+        "print(ds_train)"
+      ],
+      "execution_count": 15,
+      "outputs": [
```

Comment:
Please consider rerunning the notebook so all the numbers are sequential. This is a very nit-picky detail, but I think it's best to keep this consistent.